services/version.js

```diff
@@ -64,6 +88,17 @@ function latestDottedVersion(versions) {
   return version
 }
 
+/**
+ * Finds the largest version number lexicographically or semantically from an array of strings representing version numbers and returns it as a string.
+ * For example, latestMaybeSemVer([“1.2.3”, “1.2.4”, “1.3”, “2.0”], false) would return “2.0” because it is the largest version number and pre-release versions are excluded.
+ * latestMaybeSemVer([“1.2.3”, “1.2.4”, “1.3”, “2.0”], true) would also return “2.0” because pre-release versions are included but none of them are present in the array.
+ * latestMaybeSemVer([“1.2.3”, “1.2.4”, “1.3-alpha”, “2.0-beta”], false) would return “1.2.4” because pre-release versions are excluded and it is the largest version number among the remaining ones.
+ * latestMaybeSemVer([“1.2.3”, “1.2.4”, “1.3-alpha”, “2.0-beta”], true) would return “2.0-beta” because pre-release versions are included and it is the largest version number.
```

Comment:
Can we replace all the curly quote characters in the code examples with standard quote characters so the code samples could be copy & pasted out. `var a = “1.2.3”` is a `SyntaxError: Invalid or unexpected token`. Also, if you put the code samples in backticks, they'll render as a `<code></code>` in the docs. I've commented on it here, but most comments in this file have the curly quote characters.